contents/tutorials/beta-feedback.md

```diff
@@ -0,0 +1,135 @@
+---
+title: How to collect beta feedback from beta users
+date: 2023-11-07
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+tags: ['surveys', 'feature flags']
+---
+
+The goal of a beta is to get a feature ready for a full release. This means fixing what isn’t working. Getting beta user feedback is one of the best ways to know what to fix, and PostHog makes it easy to collect.
+
+This tutorial shows you how to combine PostHog’s [early access management](/docs/feature-flags/early-access-feature-management) and [survey](/docs/surveys) features to set up a beta and get feedback on it.
+
+## Creating an early access feature
+
+To create a beta feature in PostHog, go to the [early access management](https://app.posthog.com/early_access_features) tab and click "create feature." Add a name like "new page beta," click "Save as draft," and then "Release beta." This also automatically creates a feature flag we use to control the display of the feature.
+
+On the draft page, click "implement public opt-in" and go to the [opt-in app page](https://app.posthog.com/project/apps/574). Enable the app and set "Show features button on the page" to "Yes." This adds a modal to control early access features with no extra setup.
+
+![Creating beta feature video](../images/tutorials/beta-feedback/beta.mp4)
```

Comment:
Same as the other tutorial. These videos move+zoom too quickly. Gives me vertigo. Can we slow it down?